src/components/MovieInfo.tsx

```diff
@@ -1,14 +1,53 @@
 import React from "react";
+import { BiCameraMovie } from "react-icons/bi";
+import { PiTelevisionSimple } from "react-icons/pi";
 
-const Card = () => {
+export type MovieInfoProps = {
+  data: DataChildren[];
+};
+
+export type DataChildren = {
+  Title: string;
+  Year: string;
+  imdbID: string;
+  Type: string;
+  Poster: string;
+};
```

Comment:
it's not explicitly necessary, but it is customary to specify the items of a type in lowercase

```suggestion
export type DataChildren = {
  title: string;
  year: string;
  imdbID: string;
  type: string;
  poster: string;
};
```

Reply:
I see! In this case, I don't think I can specify the types in lowercase because the API is returning the keys with the first letter as uppercase, but correct me if I'm wrong..

Reply:
> I see! In this case, I don't think I can specify the types in lowercase because the API is returning the keys with the first letter as uppercase, but correct me if I'm wrong..

There are a couple of ways to solve this,  with pure JS or with TS Generic type alias. If we go the javascript way, one of them being is to intercept the original API response, create a function that will take the original obj keys, pass them to lowercase and create a new object with those lowercased keys in order to deliver a response with lowercase keys, however, that is not practical, being the fact that objects response can have many records, which will slow things down when it comes to delivery.

Another option would be to create a generic type that will take care of interpreting the original obj keys casing via the DataChildren type alias and convert them to lowercase using TS string case operators, in this case lowercase,  to a whole new object, and then use that one as a referral for the given params types, but, at what cost would we do something like this just to keep those four keys in lowercase?